{"uuid":"09ebefd1-a327-4965-962f-57462eec082a","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["d541be27-7663-4ad4-81e7-86244e916091"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882101334,"stop":1687882101334},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882101547,"stop":1687882101547},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882101663,"stop":1687882101663},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882101738,"stop":1687882101738},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687882101751,"stop":1687882101751}],"start":1687882101221,"stop":1687882101751}